src/shared/telemetry/util.ts

```diff
@@ -90,6 +92,33 @@ export async function getUserAgent(
     return pairs.join(' ')
 }
 
+// Although this is similair to getEnvType() in authUtil files, added new.
+// auth util can be upadate to get ComputeEnv
+
+export function getComputeEnvType() {
```

Comment:
Want to make this a multiline for documentation when hovering this function

```suggestion
/**
* Although this is similair to getEnvType() in authUtil files, added new.
* auth util can be upadate to get ComputeEnv
*/
export function getComputeEnvType() {
```